app/code/community/Apruve/ApruvePayment/controllers/WebhookController.php

```diff
@@ -48,15 +48,15 @@ public function updateOrderStatusAction()
                     $this->getResponse()->setHeader("HTTP/1.1", "404", true);
                     return;
                 };
-            } elseif ($event == 'order.accepted') {
+            } elseif ($event == 'payment_term.accepted') {
                 return; // should not be triggering anything in magento
             } elseif ($event == 'order.canceled') {
                 $orderId = $entity->merchant_order_id;
                 if (! $this->_cancelOrder($orderId)) {
                     $this->getResponse()->setHeader("HTTP/1.1", "404", true);
                     return;
                 };
-            } elseif ($event == 'payment_term.accepted') {
+            } elseif ($event == 'order.accepted') {
                 $orderId = $entity->merchant_order_id;
```

Comment:
The payment_term.accepted webhook object doesn't have this field. I don't think these should be swapped at all, though.

Reply:
I had this backwards, it's wrong right now and this fixes it.